src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherDocumentPreRules.java

```diff
@@ -64,4 +96,151 @@ protected boolean checkWireTransferTabState(DisbursementVoucherDocument dvDocume
         return tabStatesOK;
     }
 
+    /**
+     * This method notifies the user if the Check Stub Text will break into more lines than those
+     * allowed in the FastTrack file format and optionally allows the user to go back and fix the issue.
+     * If the user chooses to fix the issue, it will return false, otherwise true.
+     *
+     * @param dvDocument submitted disbursement voucher document
+     * @return Returns true if the user wishes to correct the Check Stub Text, false otherwise.
+     */
+    @SuppressWarnings("deprecation")
+    protected boolean checkCheckStubTextOverflow(DisbursementVoucherDocument dvDocument) {
+        boolean checkStubTextOK = true;
+        int maxCheckStubLines = CuDisbursementVoucherDocumentPreRules.FASTTRACK_MAX_LINE_COUNT;
+
+        List<PaymentNoteText> pntList = buildNotesForCheckStubText(dvDocument.getDisbVchrCheckStubText());
+
+        if(pntList.size() > maxCheckStubLines) {
```

Comment:
If there's no need to change the value of maxCheckStubLines, then you could try using your new constant above directly.